lib/routes/account.js

```diff
@@ -538,6 +540,29 @@ module.exports = function (
           }
         }
 
+        function sendVerifyAccountEmail() {
+          // For legacy clients, behavior is to not send an email to verify their account
+          // and have the requestor send it.
+          var sendEmailIfUnverified = false
```

Comment:
IIUC, we eventually want this to default to `true`, right?  Because legacy clients do not send the email themselves, so we want to be able to do it on their behalf.